fix_includes.py

```diff
@@ -1124,20 +1124,30 @@ def _DeleteDuplicateLines(file_lines, line_ranges):
   particular, it should not cover lines that may have C literal
   strings in them.
 
+  We only delete whole move_spans, not lines within them.
+
   Arguments:
     file_lines: an array of LineInfo objects.
     line_ranges: a list of [start_line, end_line) pairs.
   """
   seen_lines = set()
   for line_range in line_ranges:
     for line_number in range(*line_range):
-      if file_lines[line_number].type in (_BLANK_LINE_RE, _COMMENT_LINE_RE):
+      line_info = file_lines[line_number]
+      if line_info.type in (_BLANK_LINE_RE, _COMMENT_LINE_RE):
+        continue
+      if line_number != line_info.move_span[0]:
         continue
-      uncommented_line = _COMMENT_RE.sub('', file_lines[line_number].line)
-      if uncommented_line in seen_lines:
-        file_lines[line_number].deleted = True
-      elif not file_lines[line_number].deleted:
-        seen_lines.add(uncommented_line)
+      span_line_numbers = range(line_info.move_span[0], line_info.move_span[1])
+      line_infos_in_span = [file_lines[i] for i in span_line_numbers]
+      uncommented_lines = [
+              _COMMENT_RE.sub('', info.line) for info in line_infos_in_span]
```

Comment:
Indentation looks off here, I think it should be four spaces in from `uncommented` on the preceding line. Or two, I can never tell with Python :). Try `pep8` if you have it/can install it, and see which it prefers (there are existing issues, but let's address those separately).

Reply:
`pep8` was no help because it just complains everywhere that you're not using 4-space indents.  But I found elsewhere in the file 4 spaces were being used similarly, so I made it 4.

Reply:
:+1: